src/test/resources/com/puppycrawl/tools/checkstyle/checks/annotation/annotationusestyle/InputAnnotationUseStyleNoParens.java

```diff
@@ -81,9 +82,11 @@ class Closing3 {
 class Example9 {}
 @AnnotationWithAnnotationValue(value = @Another)
 class Example10 {}
-@AnnotationWithAnnotationValue(@Another()) // violation
+@AnnotationWithAnnotationValue(
+         @Another())// violation 'Annotation cannot have closing parenthesis'
```

Comment:
please add new line above and place comment "// violation below ..."

Reply:
Done. Code formatting has been rewritten to original form. Violation message has been rewritten accordingly.

Reply:
mvn clean verify passed without errors/failures